### PR TITLE
Add missing documentation for System.Security.Cryptography.*

### PR DESCRIPTION
**Pull request by bartonjs**
Fixes dotnet/runtime#54868

### CODE REVIEW

**Review by carlossanlop — APPROVED** [review]
LGTM.

Who came up with the name `ChaCha`? 😆 I can't get over it.

File: xml/System.Security.Cryptography/CryptoStream.xml [hunk 1]
```diff
@@ -524,10 +524,28 @@
...
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+Copying begins at the current position in the current stream, and does not reset the position of the destination stream after the copy operation is complete.
+  
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.ArgumentNullException"><paramref name="destination" /> is <see langword="null" />.</exception>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/CryptoStream.xml:539]:
```suggestion
        <exception cref="T:System.ArgumentNullException">
          <paramref name="destination" /> is <see langword="null" />.</exception>
```

File: xml/System.Security.Cryptography/CryptoStream.xml [hunk 2]
```diff
@@ -524,10 +524,28 @@
...
+        <exception cref="T:System.ArgumentOutOfRangeException"><paramref name="bufferSize" /> is negative or zero.</exception>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/CryptoStream.xml:540]:
```suggestion
        <exception cref="T:System.ArgumentOutOfRangeException">
          <paramref name="bufferSize" /> is negative or zero.</exception>
```

File: xml/System.Security.Cryptography/CryptoStream.xml [hunk 3]
```diff
@@ -524,10 +524,28 @@
         <Parameter Name="bufferSize" Type="System.Int32" Index="1" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="destination">To be added.</param>
-        <param name="bufferSize">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="destination">The stream to which the contents of the current stream will be copied.</param>
+        <param name="bufferSize">The size of the buffer to use for reading from the underlying stream. This value must be greater than zero. The default size is 81920.</param>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/CryptoStream.xml:528]:
```suggestion
        <param name="bufferSize">The size, in bytes, of the buffer to use for reading from the underlying stream. This value must be greater than zero. The default size is 81920 bytes.</param>
```

File: xml/System.Security.Cryptography/CryptoStream.xml [hunk 4]
```diff
@@ -557,12 +575,25 @@
...
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+Copying begins at the current position in the current stream, and does not reset the position of the destination stream after the copy operation is complete.
+  
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.ArgumentNullException"><paramref name="destination" /> is <see langword="null" />.</exception>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/CryptoStream.xml:592]:
```suggestion
        <exception cref="T:System.ArgumentNullException">
          <paramref name="destination" /> is <see langword="null" />.</exception>
```

File: xml/System.Security.Cryptography/CryptoStream.xml [hunk 5]
```diff
@@ -557,12 +575,25 @@
...
+        <exception cref="T:System.ArgumentOutOfRangeException"><paramref name="buffersize" /> is negative or zero.</exception>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/CryptoStream.xml:593]:
```suggestion
        <exception cref="T:System.ArgumentOutOfRangeException">
          <paramref name="buffersize" /> is negative or zero.</exception>
```

**Review by gewarren — APPROVED** [review]

**Comment by vcsjones** [comment]
> Who came up with the name ChaCha?

It's the successor to the Salsa20.

**Pull request merged**